### PR TITLE
Add simple documentation for prometheus-to-sd.

### PR DESCRIPTION
**Pull request by loburm**
Simple README documentation and example .yaml file for deploying
prometheus-to-sd as a side container for kube-state-metrics.

fix https://github.com/kubernetes/contrib/issues/2582

### CODE REVIEW

File: prometheus-to-sd/README.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
+# Overview
+
+prometheus-to-sd is a simple component that can scrape metrics from one or
```
Comment (piosz) [inline prometheus-to-sd/README.md]: In which format?

File: prometheus-to-sd/README.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+through the flag `source` in the next format:
+`component-name:http://host:port?whitelisted=a,b,c`. If whitelisted part is
+omitted, then all metrics that are scraped from the component will be pushed
+to the Stackdriver.
+
+## Custom metrics
+
+To be able to push custom metrics to the Stackdriver flag `metrics-prefix=custom.googleapis.com`
+has to be specified. In such case metric `bar` from the component
+`foo` is going to be pushed to the Stackdriver as `custom.googleapis.com/foo/bar`.
```
Comment (piosz) [inline prometheus-to-sd/README.md]: we should consider making it the default

File: prometheus-to-sd/README.md (hunk 1 above)
Comment (loburm) [inline prometheus-to-sd/README.md]: Added link to prometheus documentation page.

File: prometheus-to-sd/README.md (hunk 2 above)
Comment (loburm) [inline prometheus-to-sd/README.md]: Yeah it may make sense. I'll change it in the next prometheus-to-sd version.

**Comment by loburm** [comment]
@piosz could you look once again?

**Pull request merged**